core/2d/CCCamera.cpp

```diff
@@ -42,7 +42,37 @@ Viewport Camera::_defaultViewport;
 Camera* Camera::create()
 {
     Camera* camera = new Camera();
-    camera->initDefault();
+
+    auto& size = Director::getInstance()->getWinSize();
```

Comment:
```suggestion
    auto& size = _director->getWinSize();
```

Reply:
`_director` cannot be used in a static function.